rcl_action/test/rcl_action/test_action_server.cpp

```diff
@@ -135,6 +168,14 @@ TEST(TestActionServerInitFini, test_action_server_init_fini)
   ret = rcl_action_server_fini(&action_server, &node);
   EXPECT_EQ(ret, RCL_RET_OK) << rcl_get_error_string().str;
 
+  // Setup again for other finalize checks
```

Comment:
It looks like this case is already covered by the by the initialization at L143 and the fini at L168, no?